etc/config/csharp.amazon.properties

```diff
@@ -0,0 +1,17 @@
+compilers=&csharp
+supportsBinary=false
+needsMulti=false
+compilerType=csharp
+defaultCompiler=dotnet601csharp
+
+group.csharp.compilers=dotnet601csharp
+
+compiler.dotnet601csharp.exe=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked/.dotnet/dotnet
+compiler.dotnet601csharp.name=.NET 6.0.101
+compiler.dotnet601csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.1/bin/coreclr/Linux.x64.Checked
+compiler.dotnet601csharp.runtimeId=linux-x64
+compiler.dotnet601csharp.targetFramework=net6.0
+compiler.dotnet601csharp.buildConfig=Release
+compiler.dotnet601csharp.additionalSources=
+compiler.dotnet601csharp.langVersion=latest
+compiler.dotnet601csharp.nugetPackages=/opt/compiler-explorer/dotnet_nuget-v6.0.0/packages
```

Comment:
this new path contains pre-downloaded nuget packages for dotnet core, aspnet core and fsharp core